src/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="Update your description here" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<title>About Us</title>
+</head>
+
+<body>
+  <div class="topnav" id="nav">
```

Comment:
Consider using a `<nav>` tag here. Since you and Nick are both working on pages that will have this nav, you may want to coordinate and make sure the code is consistent on both pages.